cmake/CMakeLists.txt

```diff
@@ -105,7 +105,12 @@ list(APPEND SOURCES ${CMAKE_CURRENT_BINARY_DIR}/common_res.o)
 
 add_library(${PROJECT_NAME} SHARED ${SOURCES} $<TARGET_OBJECTS:wxWidgets> $<TARGET_OBJECTS:expat> $<TARGET_OBJECTS:crypto>)
 
-target_link_libraries(${PROJECT_NAME} wininet version rpcrt4 comctl32 crypt32 wsock32 ws2_32 UxTheme Shlwapi)
+if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+    # LLVM MinGW uses slightly different naming for UxTheme and Shlwapi.
+    target_link_libraries(${PROJECT_NAME} wininet version rpcrt4 comctl32 crypt32 wsock32 ws2_32 uxtheme shlwapi "${WEBVIEW2_LOADER_LIB}")
+else(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+    target_link_libraries(${PROJECT_NAME} wininet version rpcrt4 comctl32 crypt32 wsock32 ws2_32 UxTheme Shlwapi)
+endif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```

Comment:
Doesn't look like something that should matter on case-insensitive Windows filesystems?

Reply:
It matters when cross-compiling using MinGW (or similar) from Linux host.

I don't remember what project was that, but the simplest solution we've used is to use the MinGW syntax (all lower-case) which works in all cases.

Reply:
I went ahead and simplified so we use lower-case for both Clang and not-Clang.